pkg/project/admission/lifecycle/admission.go

```diff
@@ -91,6 +91,10 @@ func (e *lifecycle) Admit(a admission.Attributes) (err error) {
 
 	namespace, err := projects.GetNamespaceObject(a.GetNamespace())
 	if err != nil {
+		// do not leak that the namespace was not found
+		if apierrors.IsNotFound(err) {
+			return admission.NewForbidden(a, nil)
```

Comment:
> Is there some preferred text to spit out here?

I think its more appropriate to allow any kind of access review to be created regardless of whether the namespace exists or not.  The question of who the authorizer thinks has access to perform an action or whether the authorizer allows a subject can perform an action can be answered even if the namespace does not exist.